org.eclipse.xtext.ui.testing/src/org/eclipse/xtext/ui/testing/AbstractQuickfixTest.java

```diff
@@ -71,6 +72,24 @@ public IXtextDocument setDocument(IXtextDocument document) {
 
 	protected static class Quickfix {
 
+		@Override
```

Comment:
Could you also please keep the location of these methods below `getResult`?

Reply:
sure